dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V6.0_2015.03.07__hibernate_migration.sql

```diff
@@ -193,7 +193,7 @@ ALTER TABLE bundle2bitstream ALTER COLUMN bitstream_id SET NOT NULL;
 ALTER TABLE bundle2bitstream DROP COLUMN bundle_legacy_id;
 ALTER TABLE bundle2bitstream DROP COLUMN bitstream_legacy_id;
 ALTER TABLE bundle2bitstream DROP COLUMN id;
-ALTER TABLE bundle2bitstream add primary key (bitstream_id,bundle_id);
+ALTER TABLE bundle2bitstream add primary key (bitstream_id,bundle_id,bitstream_order);
```

Comment:
Just a note really, but making this change in an old migration will mean that we'll need developers (Committers really) to start with a "fresh" database and re-run this migration.

It's not a big deal since it'll only affect a handful of us. But, once this is merged, we'll want to inform everyone to drop their existing pre-6.0 databases and start fresh again.